channels/ke.m3u

```diff
@@ -1,17 +1,21 @@
+https://tvsn-i.akamaihd.net/hls/live/261837/tvsn/tvsn_750.m3u8
 #EXTINF:-1 tvg-id="Ebru TV Kenya" tvg-name="Ebru TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/download-1.jpg" group-title="",Ebru TV Kenya
 https://stream-01.dc3.dailymotion.com/sec(yMMZhXcH3F9vUXdVEAQnDLr2SM0UWwYkuLICgaGcRKY)/dm/3/x67n3k1/s/live-2.m3u8
-#EXTINF:-1 tvg-id="Inooro TV Kenya" tvg-name="Inooro TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/InooroTV-1.jpg" group-title="",Inooro TV Kenya
+#EXTINF:-1 tvg-id="Inooro TV Kenya" tvg-name="Inooro TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/InooroTV-1.jpg" group-title="",Inooro TV Kenya [Expiring Token]
 https://stream-01.ix7.dailymotion.com/sec(IbeXB-fEZXft8Mu82uFOZQ89_KtucjBJ-iNVupUhcks)/dm/3/x7ttbvq/s/live-4.m3u8
+#EXTINF:-1 tvg-id="K24 Kenya" tvg-name="K24" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/K24-1.jpg" tvg-url="" group-title="",K24 Kenya
+https://5f4db0f94b000.streamlock.net/k24/myStream/chunklist_w333466763.m3u8
 #EXTINF:-1 tvg-id="K24 Kenya" tvg-name="K24" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/K24-1.jpg" group-title="",K24 Kenya
 https://stream-06.ix7.dailymotion.com/sec(uK1M-ag2Z_MzR7ZEmmKyDXI_wCqDQusqgardAY5XCr0)/dm/3/x6lvncs/d/live-3.m3u8
-#EXTINF:-1 tvg-id="Kameme TV Kenya" tvg-name="Kameme TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/ke-kameme-tv-1383.jpg" group-title="",Kameme TV Kenya
+#EXTINF:-1 tvg-id="Kameme TV Kenya" tvg-name="Kameme TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/ke-kameme-tv-1383.jpg" group-title="",Kameme TV Kenya [Expiring Token]
 https://stream-05.dc3.dailymotion.com/sec(nitAgzxq_qCa0CId9XzFH2v-wTlR6kxRn5k_zLyiuaA)/dm/3/x6ol8sj/d/live-3.m3u8
-#EXTINF:-1 tvg-id="KBC Kenya" tvg-name="KBC" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/KBC-1.jpg" group-title="",KBC Kenya
+#EXTINF:-1 tvg-id="KBC Kenya" tvg-name="KBC" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/KBC-1.jpg" group-title="",KBC Kenya [Expiring Token]
 https://stream-01.dc3.dailymotion.com/sec(86G48EQSWKUUFPhHXEV5xmRLVrEbPYAHOZrsRqUifEc)/dm/3/x74211t/s/live-2.m3u8
-#EXTINF:-1 tvg-id="KTN Kenya" tvg-name="KTN Kenya" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/KTN-1.jpg" group-title="",KTN Kenya
+#EXTINF:-1 tvg-id="KTN Kenya" tvg-name="KTN Kenya" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/KTN-1.jpg" group-title="",KTN Kenya [Expiring Token]
```

Comment:
Unsupported label

Reply:
It's time to have it then. This particular channel is geoblocked and has to be updated manually